template/web/src/lib/sanityClient.js

```diff
@@ -0,0 +1,17 @@
+import sanityClient from '@sanity/client'
+import  imageUrlBuilder from '@sanity/image-url'
+import { api } from '../../../studio/sanity.json'
```

Comment:
Brilliant usage of `sanity.json` ✨

Vercel would handle this okay, right?

Reply:
I'm not sure - I assume so, but I haven't used Vercel a ton! I just saw this method on the sapper template and liked it!